lib/infra_conversion_throttler.rb

```diff
@@ -27,7 +27,11 @@ def self.start_conversions
         _log.debug("-- Available slots in EMS: #{slots}")
         _log.debug("- Looking for a conversion host for task for #{vm_name}")
 
-        eligible_hosts = ems.conversion_hosts.select(&:eligible?).sort_by { |ch| ch.active_tasks.count }
+        eligible_hosts = ems.conversion_hosts.select do |ch|
+          ch.eligible? && (!job.migration_task.warm_migration? || (job.migration_task.warm_migration? && ch.vddk_transport_supported))
```

Comment:
Could this be encapsulated in a method on `ConversionHost`?

Something like:
```
def warm_migration_eligible?
  eligible? && vddk_transport_supported
end
```

So you can do
```
eligibility = job.migration_task.warm_migration? ? :warm_migration_eligible? : :eligible?
eligible_hosts = ems.conversion_hosts.select(&eligibility).sort_by { |ch| ch.active_tasks.count }
```

Reply:
The `something && (!something_else || (another_thing && and_you_lost_me))` isn't intuitive :laughing: